README.rst

```diff
@@ -25,45 +25,44 @@ What is this plugin?
 
 A Perseus renderer wrapper for Kolibri that can track learning progress and save to the database.
 
-How to publish to PyPi?
+How can I install this plugin?
 ------------------------------
 
-1. Download this repo.
-2. Terminal move to the root level of repo dir and run the following commands:
-
-    ``git submodule update --init --recursive``
-
-    ``./update_perseus.sh``
-
-3. update `setup.py` to a newer version.
-4. Terminal move to `exercise_perseus_render` folder and run ``npm install``.
-5. Terminal move to the root level of repo dir and run the following commands to publish to PyPi:
+1. Inside your Kolibri virtual environment:
+    ``pip install kolibri-perseus-exercise-plugin``
 
-    ``python setup.py register -r pypi``
+2. Activate the plugin:
 
-    ``python setup.py sdist upload -r pypi``
+    ``kolibri plugin exercise_perseus_renderer enable``
 
+3. Restart Kolibri.
 
-How can I install this plugin?
+How can I install this plugin for development?
 ------------------------------
 
 1. Download this repo.
-2. Terminal move to the repo dir and run the following commands:
+2. Open terminal in your Kolibri repo.
+4. run the following commands:
 
-    ``git submodule update --init --recursive``
+    ``pip install -e <KOLIBRI-PERSEUS-PLUGIN-LOCAL-PATH>``
 
-    ``./update_perseus.sh``
+    ``kolibri plugin exercise_perseus_renderer enable``
 
-3. Terminal move to your Kolibri dir
-4. run the following commands:
+5. Then run the commands to install frontend packages in Kolibri, this plugin will have its recursively installed.
```

Comment:
Right, will make an edit to the `update_perseus.sh` to make it clear that has to be run on first download too.